apps/src/templates/instructions/DocumentationTab.jsx

```diff
@@ -1,26 +1,101 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
 import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
 import {useFetch} from '@cdo/apps/util/useFetch';
+import ProgrammingClassOverview from '../codeDocs/ProgrammingClassOverview';
+import {TextLink} from '@dsco_/link';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const DEFAULT_CLASS = 'Java Basics - The main Method';
 
-/**
- * This is a placeholder for the upcoming "Documentation" tab in Javalab.
- * See related task: https://codedotorg.atlassian.net/browse/CSA-361
- */
 const UnconnectedDocumentationTab = function({programmingEnvironment}) {
```

Comment:
Does this component need any more tests?

Reply:
added tests!